lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.call(alderaan);
```

Comment:
Correct. 

Side note: 
Though it was not explicitly specified that `fireOnAlderaan` should return a value, we know that all `fire` does is return a value, so we would probably also want to return that value from `fireOnAlderaan`.